C/TowerOfHanoi/README.md

```diff
@@ -0,0 +1,22 @@
+Explanation:
```

Comment:
Please don't create separate folder for this algorithm instead create ```<language_file>/Recursion/<program_file>```.
Ex: C/Recursion/TowerOfHanoi.c

And you can add README.md content as comment inside your program file